app/views/jobseekers/job_applications/build/qualifications.html.slim

```diff
@@ -14,14 +14,39 @@
     p.govuk-body = t(".description")
 
     - if job_application.qualifications.any?
-      / = render Jobseekers::JobApplications::DetailComponent.with_collection(job_application.references.includes(:job_application), title_attribute: "name", info_to_display: qualifications_info)
+      - job_application.qualification_groups.each do |group|
+        = render DetailComponent.new title: group.first.title_for_group do |detail|
+          - detail.body do
+            = govuk_summary_list classes: "govuk-!-margin-bottom-0" do |c|
+              - if group.first.secondary?
+                - c.slot(:row,
+                  key: t("jobseekers.job_applications.qualifications.subjects_and_grades"),
+                  value: group.map { |qual| sanitize([qual.subject, qual.grade].join(" – ")) }.join("</br>").html_safe)
```

Comment:
Rather than `</br>` elements, can you use `safe_join` to render them in spans/divs?